### ISSUE TITLE
문의드립니다.

### ISSUE THREAD

**Issue by hidong5** [dlimpid/nims-academy-2019-02#2]
![2222222](https://user-images.githubusercontent.com/41825429/53286550-d42e7980-37b3-11e9-8d1f-791c712a3e43.JPG)

이렇게 뜨는데 왜그러는지 알 수 있을까요?

**Comment by dlimpid** [comment]
안녕하세요. 혹시 파일 탐색기로 C:\Users(아니면 사용자)\hdo\Anaconda3\Scripts에 들어갔을 때 activate.bat라는 파일이 있나요?

**Comment by hidong5** [comment]
![default](https://user-images.githubusercontent.com/41825429/53287385-c29e9f00-37be-11e9-9d19-d54bfd6df618.JPG)

안녕하세요. 이 사진에 보시는 것처럼 그런폴더가 없어서 저도 이런경우가 처음이라  문의드렸어요 ㅠㅠ

**Comment by hidong5** [comment]
![default](https://user-images.githubusercontent.com/41825429/53287551-2cb84380-37c1-11e9-8f0b-592038b325ef.JPG)

아나콘다를 찾긴했는데 script에 activate.bat이라는 파일은 없습니다..
![default](https://user-images.githubusercontent.com/41825429/53287558-42c60400-37c1-11e9-845c-4950448d3b3b.JPG)

**Comment by dlimpid** [comment]
혹시 이전에 Windows용 Python이나 Anaconda, Miniconda를 설치하셨던 적이 있나요?

그리고 두 번째 그림에서 찾으신 Anaconda3 디렉터리는 어디에 있는 건가요?

**Comment by hidong5** [comment]
네. 이전에 파이썬 설치한적 있습니다. 현재는 지웠는데 그게 문제거 된걸까요? 두번째 그림은   c드라이브 검색에 anaconda3를 검색해본거에요!

**Comment by dlimpid** [comment]
검색해서 나온 Anaconda3가 어느 위치인지(예를 들어 C:\Users\myid\에 있는지, 아니면 C:\Program Files\에 있는지 등...)를 알려 주시면 조금 더 왜 안되는지를 아는 데 도움이 될 것 같습니다.

이전에 Python을 설치한 적이 있으시면 그게 문제일 가능성이 가장 클 것 같습니다.

1. 제어판에서 기존 Python, Anaconda 등을 전부 삭제
2. <https://java.com/ko/download/help/path.xml>을 참조하셔서 시스템 변수 설정에 들어가셔서, 사용자 변수나 시스템 변수에 PYTHONPATH가 있으면 변수 자체를 삭제
3. PATH라는 변수에 Python, Conda, Anaconda 관련 항목을 찾아서 항목만 삭제(PATH 변수 자체를 지우시면 압됩니다.)
4. C:\Users\myid\ 디렉터리에 Anaconda3, Miniconda3, .conda, .jupyter 등 디렉터리가 남아있으면 삭제
5. C:\Python2.7, C:\Python3.6 등, 기존에 Python을 설치했던 디렉터리에 남아있으면 삭제

하시고, 재부팅한 후에 다시 설치해 보시기 바랍니다.

(참고: 시스템 변수 설정은 [Rapid Environment Editor](https://www.rapidee.com/) 등의 프로그램을 이용하면 좀 더 쉽게 수정하실 수 있습니다.)
